main/cmds.go

```diff
@@ -92,22 +94,37 @@ func migrateToMostRecentSequence(ctx *log.Context, h HandlerEnvironment, seqNum
 	// do not have invent another method.  The CustomScript extension should have been using this
 	// from the beginning, but it was not.
 	//
-	computedSeqNum, err := FindSeqNumStatus(h.HandlerEnvironment.StatusFolder)
+	ctx.Log("event", "migrate to mrseq", "message", fmt.Sprintf("cannot find mrseq at %v", mostRecentSequence))
+	fout, err := os.Create(mostRecentSequence)
+	defer fout.Close()
 	if err != nil {
```

Comment:
If there is a file creation error at this point, it is very likely all subsequent operations with fout would fail, or even throw a null reference panic. We used to exit early here. Why the change to continue after os.Create(mostRecentSequence) failed?